index.php

```diff
@@ -101,16 +101,27 @@ function mysort($a, $b)
                                 continue;
                             }
 
-                            $dl = '';
-                            if (file_exists(SHELL_ROOT . 'public/' . substr($file, 5, -5) . '.sav')) {
-                                $dl = ' (DL)';
-                            }
+                            // Create savefile name from map file
+                            $saveFile = substr($file, 0, -5) .'.sav';
 
-                            echo
-                                '<tr>
-                                <!-- Owner of Map -->
-                                <td><a href="'.WWW_ROOT.'maps/' . $file. '">' . substr($file, 0, -5) . '</a></td>
+                            // Check to see if savefile exists in public folder and create download link
+                            if (file_exists(SHELL_ROOT.'saves/public/' . $saveFile)) {
 
+                              $uploaded = filemtime(SHELL_ROOT.'saves/public/'.$saveFile); // Checks the timestamp of saves in the public folder
+                              $timediff = time() - $uploaded; // Measure difference between current time and save file creation time
+                              $expired = round($timediff/1000/60,1); // Formula to show how many days and round to 2 decimal places
+
```

Comment:
wtf?
times are measured in seconds.
60*60*24*2 (seconds * minutes * hours * days) = 172800 (two days) - please remove this line